tika/server.go

```diff
@@ -41,7 +41,7 @@ type Server struct {
 	port      string
 	cmd       *exec.Cmd
 	child     *ChildOptions
-	javaprops map[string]string
+	Javaprops map[string]string
```

Comment:
Probably `JavaProps`? Also, please add a documentation comment for this field. What is it for, how is it passed to the server, etc.